certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/CodeListToBoolean.java

```diff
@@ -0,0 +1,14 @@
+package se.inera.intyg.certificateservice.domain.certificate.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class CodeListToBoolean implements ElementValue {
```

Comment:
To handle the mapping as a "fictive" ElementValue is little of a "hack". But it would require some refactoring to get rid of it. I suggest we add a refactor-task that deals with this but leave it as implemented in this PR.